### PR TITLE
Design client list page amount section and resolve horizontal scrolling issue in client list table

### PR DESCRIPTION
**Pull request by sanjibroy360**
## What
- Added the amount section.
- Resolved horizontal scrolling issue(iPad view) in the client list table.
- Designed the search box.
- Made some basic UI changes for iPad view.

## Why
- Enhancement & requirement.

## Preview
[https://youtu.be/_-2j5pvpt0I](https://youtu.be/_-2j5pvpt0I)

### CODE REVIEW

File: app/javascript/src/common/AmountBox/MobileAmountBox.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { minToHHMM } from "../../helpers/hhmmParser";
```
Comment (prasanthchaduvula) [inline app/javascript/src/common/AmountBox/MobileAmountBox.tsx:3]: Directly import from helpers `import { minToHHMM } from "helpers";`

File: app/javascript/src/components/Clients/List/index.tsx [hunk 2]
```diff
@@ -235,18 +236,20 @@ const Clients = ({ isAdminUser }) => {
   ];
 
   const currencySymb = currencySymbol(overdueOutstandingAmount?.currency);
+  const formattedOverdueAmount =
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Clients/List/index.tsx:239]: destructure `overdueOutstandingAmount` with nullish operator

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/common/AmountBox/MobileAmountBox.tsx (hunk 1 above)
Comment (sanjibroy360) [reply app/javascript/src/common/AmountBox/MobileAmountBox.tsx]: Done ✅

File: app/javascript/src/components/Clients/List/index.tsx (hunk 2 above)
Comment (sanjibroy360) [reply app/javascript/src/components/Clients/List/index.tsx]: Done ✅ 

cc: @prasanthchaduvula

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**